testing/ef_tests/src/handler.rs

```diff
@@ -971,8 +1032,11 @@ impl<E: EthSpec> Handler for KZGVerifyCellKZGProofBatchHandler<E> {
         "verify_cell_kzg_proof_batch".into()
     }
 
-    fn is_enabled_for_feature(&self, feature_name: FeatureName) -> bool {
-        feature_name == FeatureName::Fulu
+    fn is_enabled_for_feature(&self, _feature_name: FeatureName) -> bool {
```

Comment:
I think we can remove these impls for the KZG handlers as the default impl returns `false`